card_generator/tasks/cards.py

```diff
@@ -109,14 +142,14 @@ def merge_cards(self, batch_id: int) -> None:
         )
     except Exception as e:  # noqa Lets catch all errors error for debugging and retry
         logger.info(f"Error raised on client. {str(e)}")
-        self.retry(exc=e, countdown=30)
-        return
+        raise self.retry(exc=e, countdown=30)
```

Comment:
@renceInbox please make the retry interval (`countdown`) a setting provided through an environment variable.